policyengine_us/variables/gov/states/co/hcpf/chp/copay_savings/co_chp_lab_saving.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class co_chp_lab_saving(Variable):
+    value_type = float
+    entity = Person
+    label = "Colorado Child Health Plan Plus lab and imaging expense reduction"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        income_level = person("medicaid_income_level", period)
+        is_pregnant = person("is_pregnant", period)
+        p = parameters(period).gov.states.co.hcpf.chp.copays.lab
+        copay = where(is_pregnant, 0, p.calc(income_level))
+        lab_expense = person("lab_expense", period)
+        imaging_expense = person("imaging_expense", period)
```

Comment:
could we split out imaging_saving as a separate variable for consistency with others?